### ISSUE TITLE
Catch library not being pulled

### ISSUE THREAD

**Issue by ssbanerje** [ruslo/hunter#843]
I am trying to install the `Catch` library through Hunter. I initialize hunter like so -

```cmake
include("cmake/HunterGate.cmake")
HunterGate(
    URL "https://github.com/ruslo/hunter/archive/v0.16.15.tar.gz"
    SHA1 "6974c2150fc0d3b09de3ad1efcbf15d360647ffa"
)
```

And then add the Catch library like so -

```cmake
hunter_add_package(Catch)
find_package(Catch CONFIG REQUIRED)
```

This triggers an internal error saying `Catch` was not found.

```
[hunter ** INTERNAL **] Project 'Catch' not found
[hunter ** INTERNAL **] [Directory:/Users/subho/Workshop/symphony/symphony]

------------------------------ WIKI -------------------------------
    https://github.com/ruslo/hunter/wiki/error.internal
-------------------------------------------------------------------

CMake Error at /Users/subho/.hunter/_Base/Download/Hunter/0.16.15/6974c21/Unpacked/cmake/modules/hunter_wiki.cmake:12 (message):
Call Stack (most recent call first):
  /Users/subho/.hunter/_Base/Download/Hunter/0.16.15/6974c21/Unpacked/cmake/modules/hunter_internal_error.cmake:13 (hunter_wiki)
  /Users/subho/.hunter/_Base/Download/Hunter/0.16.15/6974c21/Unpacked/cmake/modules/hunter_add_package.cmake:45 (hunter_internal_error)
  symphony/CMakeLists.txt:56 (hunter_add_package)
```

I am following steps outlined [here](https://github.com/ruslo/hunter/wiki/pkg.catch). So I am not sure if I am doing something wrong, or this is a genuine bug.

**Comment by ruslo** [comment]
> I am following steps outlined here.

Version is too low: "Available since v0.17.6"

You're using 0.16.5:
```
URL "https://github.com/ruslo/hunter/archive/v0.16.15.tar.gz"
```

**Comment by ssbanerje** [comment]
That seems to have worked.
